crates/loam-cli/src/commands/dev/mod.rs

```diff
@@ -142,7 +139,8 @@ impl Cmd {
             .manifest_path
             .parent()
             .unwrap_or_else(|| Path::new("."));
-        let env_toml_path = workspace_root.join("environments.toml");
+        let env_toml_dir = workspace_root;
+        let env_toml_filename = "environments.toml";
```

Comment:
Why is this a variable? Perhaps move this to the top level as `const ENV_FILE: &str = `